server/src/internalClusterTest/java/org/opensearch/search/sort/FieldSortIT.java

```diff
@@ -1145,7 +1145,7 @@ public void testSortMissingNumbersMinMax() throws Exception {
             .get();
         assertNoFailures(searchResponse);
 
-        assertThat(searchResponse.getHits().getTotalHits().value(), equalTo(2L));
+        assertThat(searchResponse.getHits().getTotalHits().value(), equalTo(3L));
```

Comment:
Adding @harshavamsi coming from https://github.com/opensearch-project/OpenSearch/pull/17772/files#diff-f1c14da5cadfa0c16e378c5a3508bb58537a8184dbd32c22af1a10ff16e5bfde, looks like the test was modified to return 2L, but it should return 3L. With this PR the test expectations are correct and reverting to the original behavior.

Reply:
I see with this change now the tests return as expected, with this change now the query execution preserves the expected `match_all` behavior - matching ALL documents regardless of whether they have the sort field.